source/includes/_tipser-elements.md

```diff
@@ -236,4 +236,19 @@ for products on sale.
     z-index: 10;
 }
 ```
-The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
\ No newline at end of file
+The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
+<br><br><br><br><br>
+
+### Primary color ###
+```css
+.te-primary-fill {
+    background-color: #5F9F9F;
+}
+
+.te-primary-border {
+    border-color: #5F9F9F;
+}
+```
+If you'd like to unify our design with your own color-theme, you can use our primary-color theme to change the color of `add-to-cart` buttons in Product and indicator of items number in Cart. These elements on the page will be altered with one small change in your CSS:
```

Comment:
Are you sure the only thing affected by primary-color setting is add to cart button?

Reply:
Yes. Actually all buttons should be affected by this. Also e.g. `Checkout` button on `AddedToCartPopup`.

Reply:
My proposition is:

(...) theme to change the color of all buttons such as "Add to cart" in Product, but also indicator of items number in Cart.

Reply:
@piotr-sobczyk currently the only buttons are `add to cart` but what @chmurson proposed seems legit.

Reply:
@piotr-sobczyk do you have some other elements in mind to be affected by primary color? I might have omitted sth.